dev/tests/api-functional/testsuite/Magento/GraphQl/Store/StoreConfigResolverTest.php

```diff
@@ -0,0 +1,95 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\GraphQl\Store;
+
+use Magento\Store\Api\Data\StoreConfigInterface;
+use Magento\Store\Api\StoreConfigManagerInterface;
+use Magento\Store\Api\StoreRepositoryInterface;
+use Magento\Store\Api\StoreResolverInterface;
+use Magento\TestFramework\ObjectManager;
+use Magento\TestFramework\TestCase\GraphQlAbstract;
+
+/**
+ * Test the GraphQL endpoint's StoreConfigs query
+ */
+class StoreConfigResolverTest extends GraphQlAbstract
+{
+
+    /** @var  ObjectManager */
+    private $objectManager;
+
+    protected function setUp()
+    {
+        $this->objectManager = \Magento\TestFramework\Helper\Bootstrap::getObjectManager();
+    }
+
+    /**
+     * @magentoApiDataFixture Magento/Store/_files/store.php
+     */
+    public function testGetStoreConfig()
+    {
+        /** @var  StoreConfigManagerInterface $storeConfigsManager */
+        $storeConfigsManager = $this->objectManager->get(StoreConfigManagerInterface::class);
+        /** @var StoreResolverInterface $storeResolver */
+        $storeResolver = $this->objectManager->get(StoreResolverInterface::class);
+        /** @var StoreRepositoryInterface $storeRepository */
+        $storeRepository = $this->objectManager->get(StoreRepositoryInterface::class);
+        $storeId = $storeResolver->getCurrentStoreId();
+        $store = $storeRepository->getById($storeId);
+        /** @var StoreConfigInterface $storeConfig */
+        $storeConfig = current($storeConfigsManager->getStoreConfigs([$store->getCode()]));
+        $query
+            = <<<QUERY
+{
+  storeConfig{
+      items{
```

Comment:
Can you please align test. Looks like you removed "items" on your last code update. Should be also removed here.